assets/app/views/AddSiteView.js

```diff
@@ -0,0 +1,41 @@
+var fs = require('fs');
+
+var Backbone = require('backbone');
+var _ = require('underscore');
+
+var SiteModel = require('../models/Site').model;
+var templateHtml = fs.readFileSync(__dirname + '/../templates/AddSiteTemplate.html').toString();
```

Comment:
Have you considered using https://github.com/JohnPostlethwait/stringify for string requires? I've found that it's cleaner to just require the file and not need `fs` or `__dirname`. Your call though.